sim/common/sod/item_effects/phase_4.go

```diff
@@ -33,6 +34,9 @@ func init() {
 	// https://www.wowhead.com/classic/item=228139/fist-of-the-firesworn
 	// Chance on hit: Blasts the enemy for 70 Fire damage.
 	// TODO: Proc rate assumed and needs testing
+
+	itemhelpers.CreateWeaponProcDamagePhantomStrike(Jeklik, "Jeklik's Crusher", 4.0, 24257, core.SpellSchoolPhysical, 200, 20, 0.0, core.DefenseTypeMelee)
```

Comment:
Could you also order this by name among the procs?

Reply:
I'll remove this  for a future PR